bin/setup

```diff
@@ -1,29 +1,35 @@
 #!/usr/bin/env ruby
+
 require 'pathname'
+require 'fileutils'
+include FileUtils
 
 # path to your application root.
-APP_ROOT = Pathname.new File.expand_path('../../',  __FILE__)
+APP_ROOT = Pathname.new File.expand_path('../../', __FILE__)
```

Comment:
Prefer `__dir__` method over `__FILE__` constant

Reply:
Also applies to other bin/scripts

Reply:
switched all bin/scripts to `__dir__`